AdvancedPlaybackSampleApp/app/src/main/java/com/ooyala/sample/utils/SampleVideoStreamPlayer.java

```diff
@@ -0,0 +1,175 @@
+package com.ooyala.sample.utils;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.net.wifi.WifiManager;
+import android.os.Handler;
+import android.os.Message;
+import android.view.Gravity;
+import android.view.ViewGroup;
+import android.widget.FrameLayout;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.ooyala.android.OoyalaException;
+import com.ooyala.android.OoyalaPlayer;
+import com.ooyala.android.item.Stream;
+import com.ooyala.android.player.StreamPlayer;
+import com.ooyala.android.util.DebugMode;
+
+import java.util.Set;
+import java.util.Timer;
+import java.util.TimerTask;
+
+
+/**
+ * Created by achaudhari on 9/12/16.
+ */
+public class SampleVideoStreamPlayer extends StreamPlayer {
+
+    private static final String TAG = SampleVideoStreamPlayer.class.getSimpleName();
+
+    private int DURATION;
+    private final int REFRESH_RATE = 250;
+
+    private Timer _timer;
```

Comment:
can you remove underscores?  We have been slowly enforcing the removal of underscores for variables in android